### DISCUSSION TITLE
Round Robin Private Domain Alias Creation

### DISCUSSION THREAD

**Discussion by collide-vad** [simple-login/app#697] [Feature Requests]
I host multiple domains at SimpleLogin and would like to request the ability to round robin the alias address creation using all private domains hosted at SimpleLogin. Here is what I was thinking:

Domain Hosted at SimpleLogin:
- Domain1.com
- Domain2.com
- Domain3.com
- Domain4.com

user can include multiple domains in the round robin process and when Simple login is called to auto create a alias, the round robin process will create an alias using the next domain in line and loops back to the first domain.

Thanks

**Comment by nguyenkims** [comment]
@collide-vad do you think a totally random approach would work here instead of using the round robin? A random approach is easier as it doesn't require to store the last domain used :).

**Reply by collide-vad** [reply]
I think randomness would work as well...my initial request for round robin was to more equatibily spread out the aliases over all the domains. That said, I would be totally happy with random assignment as well. Thanks for considering this request.

**Reply by nguyenkims** [reply]
Just to be sure, this random or round robin algorithm should be used when creating a random alias and not when creating a custom alias right?

**Reply by collide-vad** [reply]
Yes. Only for creating random aliases and not custom.  Thanks for double checking.

**Reply by collide-vad** [reply]
@nguyenkims any update on this feature request?  Thank you

**Reply by nguyenkims** [reply]
@collide-vad unfortunately we haven't had time to work on this feature yet.
